### PR TITLE
Working on contact form section

### PR DESCRIPTION
**Pull request by mauriciomonta**
We working on the contact form section with my code partner.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,29 +296,6 @@ <h3 class="title">
         </button>
       </form>
```
Comment (alonsofl) [inline index.html:297]: - [ ] You have done a good job implementing the form but one of the project requirements is to make all the input fields `required` so the form could not be submitted if one of the fields it's empty. I would suggest fixing this issue to meet the project requirements :raised_hands:

image

- [ ] I think you have missed the footer section with the social icons. I would suggest adding the required section to match the Figma design as required :wink:
**Don't forget the section background** :pushpin:

  Figma design:
  image

**Review by alonsofl — CHANGES_REQUESTED** [review]
Hi @mauriciomonta and @Eckybrazzz,

You have done a great job so far! :muscle:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## To highlight :dart:
- Good form implementation :heavy_check_mark:
- Your design looks great! :fire:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I would suggest that the next time you submit a PR it should contain the added/changed lines in order to get a proper review. Remember that most of the time you will be working in teams and they will want to see all the introduced changes in the PR.
[Reviewed branch](https://github.com/mauriciomonta/web_portfolio/blob/contact-form/index.html)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @mauriciomonta, 

Great work on making the changes requested by a previous reviewer, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
 - [ ] There are still some inputs in your form that are not `required`. According to the previous code reviewer and the project's requirements, the user should not be able to submit the form if one or more fields are empty.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 - [ ] [OPTIONAL]: Right now your live demo link is broken. You should consider fixing the deployment or removing the link, for now.

really close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
# STATUS: APPROVED 🟢 
Hi @mauriciomonta, 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

## Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**